spec/sequel/plugins/enum_values_spec.rb

```diff
@@ -227,6 +227,150 @@
 						-> { expect { subject }.to raise_error(NoMethodError) }
 				end
 			end
+
+			shared_context('with filter_methods environment') do
+				subject(:enum_filter) { item_class.public_send(enum_value) }
+
+				def filter_sql(column, value)
+					"SELECT * FROM \"items\" WHERE (\"#{column}\" = '#{value}')"
+				end
+			end
+
+			shared_context('with filter_methods option') do |value|
+				before do
+					item_class.plugin :enum_values, filter_methods: value
+				end
+			end
+
+			describe 'filter_methods default option' do
+				before { item_class.plugin :enum_values }
```

Comment:
I see a duplication with `with filter_methods option` shared context. I think we can use it here. For default behavior, just define something like `enum_values_options` with default like `{}`, and then `{ filter_methods: true }`.

Reply:
Done, thx for advise.